src/components/accounts/CreateAccount.js

```diff
@@ -80,34 +91,27 @@ class CreateAccount extends Component {
     }
 
     handleCreateAccount = async () => {
-        const { accountId, token } = this.state;
-        const { match, createNewAccount, setFormLoader } = this.props
-
-        const fundingKey = match.params.fundingKey;
-        const fundingContract = match.params.fundingContract;
+        const { accountId } = this.state;
+        const { 
+            setFormLoader,
+            fundingContract, fundingKey,
+        } = this.props
 
-        this.setState({ loader: true });
-        
-        try {
-            await createNewAccount(accountId, fundingKey, fundingContract, token)
-        } finally {
-            this.setState({ loader: false });
-            setFormLoader(false)
-        }
+        setFormLoader(false)
         
-        this.handleCreateAccountSuccess();
-    }
+        this.setState({ loader: true });
+
+        const linkdropParams = fundingContract ? `${fundingContract}/${fundingKey}` : ``
+        let nextUrl = process.env.DISABLE_PHONE_RECOVERY === 'yes' ?
+            `/setup-seed-phrase/${accountId}/phrase/1/${linkdropParams}` :
```

Comment:
what does `/phrase/1` mean here? Is `1` intentionally hardcoded?

Reply:
1 maps to isNew.

This is to indicate the recovery method (seed phrase) should attempt to create a new account when confirmed.

I understand we can check the account is new by checking if it already exists on chain, will do this ASAP as a fix.

Reply:
Again, not the best pattern but will tackle in a PR fix.